lib/config/env.ts

```diff
@@ -67,7 +67,7 @@ export function getConfig(env: NodeJS.ProcessEnv): RenovateConfig {
 
   if (env.DOCKER_USERNAME && env.DOCKER_PASSWORD) {
```

Comment:
Shouln't we move this to the datasource as optional api function for config initialization?

Reply:
Interesting idea